packages/terra-spacer/src/Spacer.scss

```diff
@@ -0,0 +1,325 @@
+:local {
+  .margin-top-none {
+    margin-top: 0;
+  }
+
+  .margin-top-small-minus-2 {
+    margin-top: var(--terra-spacer-margin-top-small-minus-2, 0.214rem); // 3px
+  }
+
+  .margin-top-small-minus-1 {
+    margin-top: var(--terra-spacer-margin-top-small-minus-1, 0.357rem); // 5px
+  }
+
+  .margin-top-small {
+    margin-top: var(--terra-spacer-margin-top-small, 0.5rem); // 7px
+  }
+
+  .margin-top-medium {
+    margin-top: var(--terra-spacer-margin-top-medium, 0.714rem); // 10px
```

Comment:
Update `0.714rem` to `0.7142857143rem`. The extra decimal values help ensure the rem computes to a whole value, 10px, instead of a sub-pixel value.

Reply:
Updated https://github.com/cerner/terra-core/pull/1011/commits/b2ec65a5ad9f9f5b81c9394939eb477f56b21774#diff-bbd694561d6e058afb7ec7501feecb1aR1